# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/vr/service/OSVRSession.cpp
# Commit: 898ed02804fe
# Full Hash: 898ed02804fefc9786112f135b9621cc444e6860
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-23 15:52:15
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r e1b7abc99ae9 -r 898ed02804fe gfx/vr/service/OSVRSession.cpp
--- a/gfx/vr/service/OSVRSession.cpp	Wed May 22 12:44:04 2019 +0000
+++ b/gfx/vr/service/OSVRSession.cpp	Wed May 22 22:27:37 2019 +0000
@@ -6,9 +6,9 @@
 
 #include "OSVRSession.h"
 #include "prenv.h"
-#include "gfxPrefs.h"
 #include "nsString.h"
 #include "mozilla/Preferences.h"
+#include "mozilla/StaticPrefs.h"
 #include "mozilla/SharedLibrary.h"
 #include "mozilla/gfx/Quaternion.h"
 
@@ -209,7 +209,7 @@
 OSVRSession::~OSVRSession() { Shutdown(); }
 
 bool OSVRSession::Initialize(mozilla::gfx::VRSystemState& aSystemState) {
-  if (!gfxPrefs::VREnabled() || !gfxPrefs::VROSVREnabled()) {
+  if (!StaticPrefs::VREnabled() || !StaticPrefs::VROSVREnabled()) {
     return false;
   }
   if (mOSVRInitialized) {